sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/datasketchesAggregates.scala

```diff
@@ -265,12 +271,16 @@ case class HllUnionAgg(
   override protected def withNewChildrenInternal(newLeft: Expression, newRight: Expression):
   HllUnionAgg = copy(left = newLeft, right = newRight)
 
+  // Overrides for ExpectsInputTypes
+
+  override def inputTypes: Seq[AbstractDataType] = Seq(BinaryType, BooleanType)
+
+  override def inputIsFoldable: Seq[Option[Boolean]] = Seq(None, Some(true))
```

Comment:
`None` means that you don't care of it is foldable or not, correct? You just require to be foldable for the second parameter.

Reply:
Yes, that's correct.